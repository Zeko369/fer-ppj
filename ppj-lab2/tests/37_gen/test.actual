err IDN 5 k
