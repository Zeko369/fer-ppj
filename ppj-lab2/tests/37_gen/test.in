IDN 1 n
OP_PRIDRUZI 1 =
L_ZAGRADA 1 (
IDN 1 n
D_ZAGRADA 1 )
OP_PUTA 1 *
OP_PLUS 1 +
BROJ 1 74
OP_MINUS 1 -
BROJ 1 1035
OP_PLUS 1 +
BROJ 1 432
IDN 2 h
OP_PRIDRUZI 2 =
BROJ 2 773
OP_PUTA 2 *
L_ZAGRADA 2 (
BROJ 2 1019
D_ZAGRADA 2 )
OP_MINUS 2 -
BROJ 2 6
OP_MINUS 2 -
L_ZAGRADA 2 (
IDN 2 h
OP_PLUS 2 +
L_ZAGRADA 2 (
L_ZAGRADA 2 (
BROJ 2 1072
D_ZAGRADA 2 )
OP_MINUS 2 -
BROJ 2 1026
OP_MINUS 2 -
BROJ 2 367
D_ZAGRADA 2 )
D_ZAGRADA 2 )
IDN 3 h
OP_PRIDRUZI 3 =
BROJ 3 1226
OP_MINUS 3 -
BROJ 3 1002
IDN 4 n
OP_PRIDRUZI 4 =
OP_PLUS 4 +
BROJ 4 1008
OP_PUTA 4 *
IDN 4 h
OP_PUTA 4 *
OP_PLUS 4 +
IDN 4 n
OP_PLUS 4 +
L_ZAGRADA 4 (
L_ZAGRADA 4 (
BROJ 4 799
OP_MINUS 4 -
L_ZAGRADA 4 (
BROJ 4 348
D_ZAGRADA 4 )
D_ZAGRADA 4 )
OP_MINUS 4 -
OP_MINUS 4 -
L_ZAGRADA 4 (
BROJ 4 742
OP_PLUS 4 +
OP_MINUS 4 -
OP_PLUS 4 +
IDN 4 n
D_ZAGRADA 4 )
D_ZAGRADA 4 )
OP_PUTA 4 *
IDN 4 h
IDN 5 k
OP_PRIDRUZI 5 =
OP_PLUS 5 +
L_ZAGRADA 5 (
BROJ 5 1120
OP_PUTA 5 *
BROJ 5 1068
D_ZAGRADA 5 )
OP_PLUS 5 +
IDN 5 k
OP_PLUS 5 +
OP_MINUS 5 -
OP_MINUS 5 -
BROJ 5 1068
OP_PUTA 5 *
BROJ 5 994
