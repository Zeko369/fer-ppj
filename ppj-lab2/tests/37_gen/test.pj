n = (n)*+74 - 1035 + 432
h = 773*(1019) - 6 - (h + ((1072) - 1026 - 367))
h = 1226 - 1002
n = +1008*h*+n + ((799 - (348)) - -(742 + -+n))*h
k = +(1120*1068) + k + --1068*994
