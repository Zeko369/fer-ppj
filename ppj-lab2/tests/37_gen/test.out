<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 n
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       L_ZAGRADA 1 (
       <E>
        <T>
         <P>
          IDN 1 n
         <T_lista>
          $
        <E_lista>
         $
       D_ZAGRADA 1 )
      <T_lista>
       OP_PUTA 1 *
       <T>
        <P>
         OP_PLUS 1 +
         <P>
          BROJ 1 74
        <T_lista>
         $
     <E_lista>
      OP_MINUS 1 -
      <E>
       <T>
        <P>
         BROJ 1 1035
        <T_lista>
         $
       <E_lista>
        OP_PLUS 1 +
        <E>
         <T>
          <P>
           BROJ 1 432
          <T_lista>
           $
         <E_lista>
          $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 h
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 773
       <T_lista>
        OP_PUTA 2 *
        <T>
         <P>
          L_ZAGRADA 2 (
          <E>
           <T>
            <P>
             BROJ 2 1019
            <T_lista>
             $
           <E_lista>
            $
          D_ZAGRADA 2 )
         <T_lista>
          $
      <E_lista>
       OP_MINUS 2 -
       <E>
        <T>
         <P>
          BROJ 2 6
         <T_lista>
          $
        <E_lista>
         OP_MINUS 2 -
         <E>
          <T>
           <P>
            L_ZAGRADA 2 (
            <E>
             <T>
              <P>
               IDN 2 h
              <T_lista>
               $
             <E_lista>
              OP_PLUS 2 +
              <E>
               <T>
                <P>
                 L_ZAGRADA 2 (
                 <E>
                  <T>
                   <P>
                    L_ZAGRADA 2 (
                    <E>
                     <T>
                      <P>
                       BROJ 2 1072
                      <T_lista>
                       $
                     <E_lista>
                      $
                    D_ZAGRADA 2 )
                   <T_lista>
                    $
                  <E_lista>
                   OP_MINUS 2 -
                   <E>
                    <T>
                     <P>
                      BROJ 2 1026
                     <T_lista>
                      $
                    <E_lista>
                     OP_MINUS 2 -
                     <E>
                      <T>
                       <P>
                        BROJ 2 367
                       <T_lista>
                        $
                      <E_lista>
                       $
                 D_ZAGRADA 2 )
                <T_lista>
                 $
               <E_lista>
                $
            D_ZAGRADA 2 )
           <T_lista>
            $
          <E_lista>
           $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 h
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         BROJ 3 1226
        <T_lista>
         $
       <E_lista>
        OP_MINUS 3 -
        <E>
         <T>
          <P>
           BROJ 3 1002
          <T_lista>
           $
         <E_lista>
          $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 n
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          OP_PLUS 4 +
          <P>
           BROJ 4 1008
         <T_lista>
          OP_PUTA 4 *
          <T>
           <P>
            IDN 4 h
           <T_lista>
            OP_PUTA 4 *
            <T>
             <P>
              OP_PLUS 4 +
              <P>
               IDN 4 n
             <T_lista>
              $
        <E_lista>
         OP_PLUS 4 +
         <E>
          <T>
           <P>
            L_ZAGRADA 4 (
            <E>
             <T>
              <P>
               L_ZAGRADA 4 (
               <E>
                <T>
                 <P>
                  BROJ 4 799
                 <T_lista>
                  $
                <E_lista>
                 OP_MINUS 4 -
                 <E>
                  <T>
                   <P>
                    L_ZAGRADA 4 (
                    <E>
                     <T>
                      <P>
                       BROJ 4 348
                      <T_lista>
                       $
                     <E_lista>
                      $
                    D_ZAGRADA 4 )
                   <T_lista>
                    $
                  <E_lista>
                   $
               D_ZAGRADA 4 )
              <T_lista>
               $
             <E_lista>
              OP_MINUS 4 -
              <E>
               <T>
                <P>
                 OP_MINUS 4 -
                 <P>
                  L_ZAGRADA 4 (
                  <E>
                   <T>
                    <P>
                     BROJ 4 742
                    <T_lista>
                     $
                   <E_lista>
                    OP_PLUS 4 +
                    <E>
                     <T>
                      <P>
                       OP_MINUS 4 -
                       <P>
                        OP_PLUS 4 +
                        <P>
                         IDN 4 n
                      <T_lista>
                       $
                     <E_lista>
                      $
                  D_ZAGRADA 4 )
                <T_lista>
                 $
               <E_lista>
                $
            D_ZAGRADA 4 )
           <T_lista>
            OP_PUTA 4 *
            <T>
             <P>
              IDN 4 h
             <T_lista>
              $
          <E_lista>
           $
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 5 k
        OP_PRIDRUZI 5 =
        <E>
         <T>
          <P>
           OP_PLUS 5 +
           <P>
            L_ZAGRADA 5 (
            <E>
             <T>
              <P>
               BROJ 5 1120
              <T_lista>
               OP_PUTA 5 *
               <T>
                <P>
                 BROJ 5 1068
                <T_lista>
                 $
             <E_lista>
              $
            D_ZAGRADA 5 )
          <T_lista>
           $
         <E_lista>
          OP_PLUS 5 +
          <E>
           <T>
            <P>
             IDN 5 k
            <T_lista>
             $
           <E_lista>
            OP_PLUS 5 +
            <E>
             <T>
              <P>
               OP_MINUS 5 -
               <P>
                OP_MINUS 5 -
                <P>
                 BROJ 5 1068
              <T_lista>
               OP_PUTA 5 *
               <T>
                <P>
                 BROJ 5 994
                <T_lista>
                 $
             <E_lista>
              $
      <lista_naredbi>
       $
